assembler/assemble.py

```diff
@@ -825,6 +825,8 @@ def args_ry(self, tokens,opcode):
     def args_rgm(self, tokens,opcode):
         #BIT BSET BCLR BTG
         arg_count_test(len(tokens),3)
+        if tokens[1] in ["RS", "IN", "OUT"]:
+            tokens[1] = "R3"
         if tokens[1] not in ["R0","R1","R2","R3"]:
             raise ParserError("E::This opcode requires R0, R1, R2, or R3 as the first argument")
```

Comment:
Maybe update the error message too.

Reply:
Yeah, fair enough. Let's resolve the `IN/OUT` aliases before I update this though.